dotnet-authserver/src/TeacherIdentity.AuthServer/Api/V1/Handlers/UpdateUserHandler.cs

```diff
@@ -82,7 +82,8 @@ public async Task<UserInfo> Handle(UpdateUserRequest request, CancellationToken
             FirstName = user.FirstName,
             LastName = user.LastName,
             Trn = user.Trn,
-            UserId = user.UserId
+            UserId = user.UserId,
+            MobileNumber = user.MobileNumber
```

Comment:
Updated here as property is required. Have not added ability to update mobile number via this endpoint as not in scope